### PR TITLE
Displaying a contextual page title

### PR DESCRIPTION
**Pull request by carpentierma**

### CODE REVIEW

File: app/Category.php [hunk 1]
```diff
@@ -14,4 +14,14 @@ public function places()
...
     }
+
+
+    /**
+     * Method returning the page title of the object
+     * @return string
+     */
+    public function getPageTitle(): string
+    {
+        return $this->name . ' - Catégorie - ' . config('app.name', '');
```
Comment (jpmurray) [inline app/Category.php]: Opening discussion here, but do you think it makes the title a bit too long?

File: app/Place.php [hunk 2]
```diff
@@ -38,4 +38,13 @@ public function getCompleteAddressAttribute()
...
         return "{$this->address}, {$this->city}, {$this->province}, {$this->postalCode}";
     }
+
+    /**
+     * Method returning the page title of the object
+     * @return string
+     */
+    public function getPageTitle(): string 
+    {
+        return $this->name . ' - Entreprise - ' . config('app.name', '');
```
Comment (jpmurray) [inline app/Place.php]: Same interogation as with categories.

File: app/Region.php [hunk 3]
```diff
@@ -14,4 +14,13 @@ public function places()
...
         return $this->hasMany(Place::class);
     }
+
+    /**
+     * Method returning the page title of the object
+     * @return string
+     */
+    public function getPageTitle(): string 
+    {
+        return $this->name . ' - Région - ' . config('app.name', '');
```
Comment (jpmurray) [inline app/Region.php]: Same interogation as with other titles :)

**Comment by jpmurray** [comment]
Nice touch the link to Google Maps. Enables people to double check !

File: app/Category.php (hunk 1 above)
Comment (poliquinp) [reply app/Category.php]: Yes. I would remove `- Catégorie -` from the title to just get the game and app name.

File: app/Place.php (hunk 2 above)
Comment (poliquinp) [reply app/Place.php]: For Place, I would go  [Place name] - [Region] - [App name]. This way we would be SEO friendly with a Page Title not longer then 70 caracters.

File: app/Region.php (hunk 3 above)
Comment (poliquinp) [reply app/Region.php]: Same structure as Category

File: app/Category.php (hunk 1 above)
Comment (carpentierma) [reply app/Category.php]: Done

File: app/Place.php (hunk 2 above)
Comment (carpentierma) [reply app/Place.php]: Done

File: app/Region.php (hunk 3 above)
Comment (carpentierma) [reply app/Region.php]: Done

**Review by jpmurray — APPROVED** [review]
Sounds good. Will test this in staging then merge to master soon enought.

**Pull request merged**